docs/General/Manuals/client-manual.rst

```diff
@@ -1111,10 +1111,160 @@ a table below the widget, such as Figure 26, that displays the amount of time un
 Cities View
 ^^^^^^^^^^^
 
+The :guilabel:`Cities View` is a separate page in a table format. When you click the icon for it as shown in
+Figure 27, the client will leave the view you are currently showing and will switch to a listing of your
+cities.
+
+.. figure:: ../../_static/images/gui-elements/top-bar-cities.png
+  :align: center
+  :alt: Freeciv21 Cities
+  :figclass: align-center
+
+  Figure 27: Cities View Icon
+
+
+Figure 28 gives a sample of the :guilabel:`City View` in the client with the default columns displayed. If
+you right-click on the table heading, you will be given a list of other columns you may want to show. If you
+change anything, then be sure to save the settings from the :guilabel:`Game` menu.
```

Comment:
Side note: shouldn't the client auto-save this?

Reply:
I don't know if it does, I just am in the habit of not assuming.